odl/solvers/util/callback.py

```diff
@@ -315,6 +316,13 @@ def __init__(self, *args, **kwargs):
         ----------
         display_step : positive int, optional
             Number of iterations between plots. Default: 1
+        saveto : string, optional
+            Path to a directory where the figures are to be saved, potentially
+            followed by a file name. If the directory name does not exist, a
+            ``ValueError`` is raised. If not given, the figures are not saved.
+            Default: None
+        suffix : string, optinal
```

Comment:
Getting the suffix from the `saveto` would mean that we have to strip the `saveto` of a potential `.myFormat`, add the iteration counts and then put it back. That seems cumbersome to me...

Regarding the format-type string: since I was not sure of how/why that works, I was not sure of how to implement this :p I still don't know how to implement that in an efficient way, if it is to include a file format too...?